busypie/awaiter.py

```diff
@@ -40,7 +43,9 @@ def _raise_exception_if_not_ignored(self, e: Exception):
             raise e
 
     def _validate_wait_constraint(self, condition_evaluator: ConditionEvaluator, start_time: float):
-        if (time.time() - start_time) > self._condition.wait_time_in_secs:
+        execute_time = (time.time() - start_time)
+        if ((hasattr(self._condition, 'at_least') and self._condition.at_least < execute_time)
```

Comment:
When you don't have "at_least" property?

Reply:
when you are not going to us it

Reply:
FIXED